src/site/_includes/components/component-preview/index.ts

```diff
@@ -72,7 +72,7 @@ export function initComponentPreviewsIframe() {
     iframeEl.id = iframeId
     iframeEl.style.height = `${componentPreviewContentItem.clientHeight}px`
     iframeEl.className = componentPreviewContentItem.className
-    iframeEl.src = `preview/${exampleName}/embed.html`
+    iframeEl.src = `/components/${componentName}/preview/${exampleName}/embed.html`
```

Comment:
I had to change the url here to fix the 404 on the error-message page. This is because the error-message component page uses the date-input component